ibu-imager/README.md

```diff
@@ -153,17 +154,17 @@ To create an IBU seed image out of your Single Node OpenShift (SNO), run the fol
  				-v /var:/var \
  				-v /var/run:/var/run \
  				-v /run/systemd/journal/socket:/run/systemd/journal/socket \
- 				quay.io/lochoa/ibu-imager:4.14.0 create --authfile /var/lib/kubelet/config.json --registry ${LOCAL_USER_REGISTRY}
+ 				quay.io/lochoa/ibu-imager:4.14.0 create --authfile /var/lib/kubelet/config.json --image quay.io/somewhere/ibu-seed-images:4.14.0-du
```

Comment:
could we also update the HACKING.md docs about this change? thanks!

https://github.com/openshift-kni/lifecycle-agent/blob/main/HACKING.md#running-the-ibu-imager

Reply:
done! :)